framework/components/AStepper/AStepper.scss

```diff
@@ -34,9 +75,7 @@ $step-line-top: 22px;
 $step-line-vertical-top: 44px;
 $step-line-left: 44px;
 $step-line-vertical-left: 22px;
```

Comment:
Can you please remove all of these scss variables? They're potential collisions, don't have proper defaults, and even if they had proper defaults we don't yet have a proper solution for documenting scss variables

Reply:
only this 3? or all of them?

Reply:
all of them